ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/custom-templates.cfg

```diff
@@ -12,6 +12,8 @@ interface Management1
    vrf MGMT
    ip address 10.73.255.122/24
 !
+end
```

Comment:
Hmm yeah this is because I moved custom-templates outside so it is rendered at the end.. not great